plenum/server/primary_elector.py

```diff
@@ -124,11 +113,6 @@ def filterMsgs(self, wrappedMsgs: deque) -> deque:
                 reqViewNo = getattr(msg, f.VIEW_NO.nm)
                 if reqViewNo == self.viewNo:
                     filtered.append(wrappedMsg)
-                elif reqViewNo > self.viewNo:
```

Comment:
Node does not pass higher view message to elector now, earlier it was the elector's responsibility to queue such messages, now its not